syntax = "proto3";

message MetaRequest {
    string full_name = 1;
    bytes request = 2;
}

message MetaResponse {
    string full_name = 1;
    bytes response = 2;
}